vcs/repository.go

```diff
@@ -140,10 +141,24 @@ type Diff struct {
 	Raw string // the raw diff output
 }
 
+// BranchesOptions specifies options for the list of branches returned by
+// (Repository).Branches.
+type BranchesOptions struct {
+	BehindAheadBranch string `json:",omitempty" url:",omitempty"` // Or "" to disable fetching behind/ahead counts.
```

Comment:
Possible alternative names are `BaseBranch`, `BehindAheadBaseBranch`. I should also elaborate on what it does exactly in the docs.